dash-pipeline/SAI/specs/dash_outbound_routing.yaml

```diff
@@ -2,6 +2,26 @@
 name: dash_outbound_routing
 description: ''
 sai_apis:
+- !!python/object:utils.sai_spec.sai_api.SaiApi
```

Comment:
the order here is also not right, do you mind to help review #577 to avoid this issue?

Reply:
Regenerated after merging master.